message_definitions/v1.0/ras_a.xml

```diff
@@ -4,6 +4,30 @@
   <version>0</version>
   <dialect>0</dialect>
   <enums>
+    <enum name="MAV_AUTOPILOT">
+      <description>Micro air vehicle / autopilot classes. This identifies the individual model.</description>
+      <entry value="21" name="MAV_AUTOPILOT_SKYDIO">
+        <description>Skydio - https://www.skydio.com/</description>
+      </entry>
+      <entry value="22" name="MAV_AUTOPILOT_SHIELD_AI">
+        <description>Shield AI - https://shield.ai/</description>
+      </entry>
+    </enum>
+    <enum name="MAV_PROTOCOL_CAPABILITY" bitmask="true">
+      <description>Bitmask of (optional) autopilot capabilities (64 bit). If a bit is set, the autopilot supports this capability.</description>
+      <entry value="262144" name="MAV_PROTOCOL_CAPABILITY_INDOOR_OBSTACLE_AVOIDANCE">
+        <description>Autopilot is capable of indoor obstacle avoidance. The obstacle visualization range (i.e. omnidirectional or single direction) is not determined by this flag and is system specific.</description>
+      </entry>
+      <entry value="524288" name="MAV_PROTOCOL_CAPABILITY_OUTDOOR_OBSTACLE_AVOIDANCE">
+        <description>Autopilot is capable of outdoor obstacle avoidance. The obstacle visualization range (i.e. omnidirectional or single direction) is not determined by this flag and is system specific.</description>
+      </entry>
```

Comment:
What do we expect that GCSes will do with this information?

Reply:
Not specifically a GCS, but an autonomy engine in the ground can use this info for appropriate planning. For example, CCAST.